packages/admin-ui/src/filters/filter-multiple/filter-multiple.tsx

```diff
@@ -22,31 +23,30 @@ export function FilterMultiple<T>(props: FilterMultipleProps<T>) {
   } = props
 
   return (
-    <BaseFilter
-      state={state}
-      appliedValuesLabel={
-        <MultipleItemsLabel appliedItems={appliedItems} state={state} />
-      }
-    >
-      {items.map((item) => (
-        <ComboboxItem
-          aria-selected={combobox.isSelected(item)}
-          key={getOptionId(item)}
-          onClick={() => combobox.onChange(item)}
-          style={style.option}
-          id={`${baseId ?? ''}-item-${getOptionId(item)}`}
-        >
-          <Checkbox
-            id={getOptionId(item)}
-            checked={combobox.isSelected(item)}
-            aria-checked={undefined}
-            csx={{ marginRight: '$s' }}
-            label={getOptionLabel(item)}
-            aria-readonly="true"
-          />
-        </ComboboxItem>
-      ))}
-    </BaseFilter>
+    <>
+      <FilterDisclosure state={state} appliedItems={appliedItems}>
+        {disclosureLabel}
+      </FilterDisclosure>
+      <BaseFilter state={state}>
+        {items.map((item) => (
+          <ComboboxItem
+            aria-selected={combobox.isSelected(item)}
+            key={getOptionId(item)}
+            onClick={() => combobox.onChange(item)}
+            style={style.option}
+            id={`${baseId ?? ''}-item-${getOptionId(item)}`}
+          >
+            <Checkbox
+              id={getOptionId(item)}
+              checked={combobox.isSelected(item)}
+              aria-checked={undefined}
```

Comment:
Out of curiosity: why is this always `undefined`?

Reply:
There is another aria check indicator above on the tree, this one would be a duplicate on the same item